components/brave_wallet/browser/keyring_controller.cc

```diff
@@ -28,6 +28,7 @@
 #include "components/prefs/scoped_user_pref_update.h"
 #include "crypto/random.h"
 #include "third_party/abseil-cpp/absl/types/optional.h"
+#include "third_party/re2/src/re2/re2.h"
```

Comment:
is this dep added to BUILD.gn?

Reply:
Yes the dep already exists for `"//third_party/re2",`